alicloud/resource_alicloud_cs_serveless_kubernetes.go

```diff
@@ -0,0 +1,309 @@
+package alicloud
+
+import (
+	"time"
+
+	"github.com/denverdino/aliyungo/common"
+	"github.com/denverdino/aliyungo/cs"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func resourceAlicloudCSServelessKubernetes() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAlicloudCSServelessKubernetesCreate,
+		Read:   resourceAlicloudCSServelessKubernetesRead,
+		Delete: resourceAlicloudCSServelessKubernetesDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
+			Update: schema.DefaultTimeout(30 * time.Minute),
```

Comment:
If there is no `Update` method, please remove `Update Timeout`.

Reply:
removed